public/gameInfo.php

```diff
@@ -962,8 +963,6 @@ function submitSetRequest(user, gameID) {
 
                             $earnedOnHardcore = isset($nextAch['DateEarnedHardcore']);
 
-                            $achDesc = str_replace('"', '\'', $achDesc);
```

Comment:
There's no reason to change double quotes to single quotes for display on the web page. You can see this behavior in the "before" image in the PR description. `"47______"` in the title, but `'47______'` in the description.